### ISSUE TITLE
invalid syntax in doc/core/examples/courier.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3599]
|<img alt="benjamin's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| benjamin reported|
|-|-|
|Trac ID|trac#3599|
|Type|defect|
|Created|2009-01-04 20:08:03Z|

It's just a simple indentation problem.

Attachments:

* [invalid_syntax_fix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d68/d68686317f39a3cb7bb2eeb9b0b112f4e5ac309b/8fa6f9f50e5272910467db17efdf5bbdee6ee305.patch) (451 bytes) - added by benjamin on 2009-01-04 20:08:42Z - patch for this issue

<details><summary>Searchable metadata</summary>

```
trac-id__3599 3599
type__defect defect
reporter__benjamin benjamin
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1231099683000000 1231099683000000
changetime__1231371036000000 1231371036000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra set owner to @therve|
|-|-|

Looks good to me, someone apply the patch before I do it.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

(In [25867]) Apply invalid_syntax_fix.patch

Author: benjamin
Reviewer: thijs
Fixes: [#3599](https://github.com/twisted/twisted/issues/3599)

Correct the indentation in this Courier mail filter example so that it is valid
Python syntax.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
